### ISSUE TITLE
`Defect.program()` should be replaced to the `Defect.object()`

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#525]
In EO, we [got rid](https://github.com/objectionary/eo/releases/tag/0.56.0) of `program` root node in XMIR. However, we still using `program` in the `org.eolang.Defect`, which creates inconsistency in the EO family. I suggest to rename both: 1) Java method in the `Defect` interface, and 2) attribute in the XML Defect representation. So this:

```xml
<defect line="42" program="foo" severity="warning">Boom!</defect>
```

Will look like this:

```xml
<defect line="42" object="foo" severity="warning">Boom!</defect>
```

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by 0crat** [comment]
@h1alexbel 🐛 Fantastic bug report! You've earned +8 points, boosting your balance to [+185](https://www.eolang.org/zerocracy/objectionary-vitals.html). Your keen eye helps elevate our project's quality. Keep spotting things that [don't look right](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) and [report them](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) without hesitation. Remember, [bugs are welcome](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) – they're opportunities for improvement!
